packages/components/atoms/f-toggle/src/components/Toggle.vue

```diff
@@ -0,0 +1,223 @@
+            return this.$attrs.checked;
+        }
+    },
+
+    created () {
+        this.uniqueIdentifierId = uuid();
+    },
+
+    methods: {
+        toggleOption (event) {
+            this.$emit('update', event.target.checked);
+        }
+    }
+};
+</script>
+
+<style lang="scss" module>
+@use '@justeat/fozzie/src/scss/fozzie' as f;
+
+$toggle-colour: f.$color-interactive-form;
+$toggle-colour--checked: f.$color-interactive-brand;
+$toggle-colour--disabled: f.$color-disabled-01;
+
+$toggle-width: 48px;
+$toggle-height: 24px;
+$toggle-control-size: 20px;
+$toggle-padding: 2px;
+$toggle-radius: f.$radius-rounded-e;
+$toggle-icon-size: 14px;
+
+$toggle-translation: translateX(calc($toggle-width - $toggle-control-size - 2 * $toggle-padding));
+
+@mixin toggle-transition ($property) {
+    transition-property: $property;
+    transition-duration: 0.15s;
+    transition-timing-function: cubic-bezier(0.4, 0, 0.9, 1);
+    transition-delay: 0s;
+}
+
+.c-toggle {
+    display: flex;
+
+    @include toggle-transition(background-color);
+
+    width: $toggle-width;
+    height: $toggle-height;
+    padding: $toggle-padding;
+    border-radius: $toggle-radius;
+    background-color: $toggle-colour;
+
+    cursor: pointer;
+
+    &:hover {
+        background-color: darken($toggle-colour, f.$color-hover-01);
+    }
+
+    &:focus,
+    &:focus-within {
+        background-color: $toggle-colour;
+        box-shadow: 0 0 0 2px f.$color-focus-inner, 0 0 0 4px f.$color-focus-outer;
+    }
+
+    &:active {
+        background-color: darken($toggle-colour, f.$color-active-01);
+    }
+}
+
+.c-toggle-input {
+    appearance: none;
+    margin: 0;
+}
+
+.c-toggle-control {
+    @include toggle-transition(transform);
+
+    display: flex;
+
+    align-items: center;
+    justify-content: center;
+
+    width: $toggle-control-size;
+    height: $toggle-control-size;
+
+    border-radius: $toggle-radius;
+    background-color: f.$color-content-interactive-light;
+}
+
+.c-toggle-icon {
+    width: $toggle-icon-size;
+    height: $toggle-icon-size;
+
+    path {
+        @include toggle-transition(fill);
+        fill: $toggle-colour--checked;
+    }
+}
+
+
+.c-toggle--checked {
+    @include toggle-transition(background-color);
+    background-color: $toggle-colour--checked;
+
+    &:hover {
+        background-color: darken($toggle-colour--checked, f.$color-hover-01);
+    }
+
+    &:focus,
+    &:focus-within {
+        background-color: $toggle-colour--checked;
+    }
+
+    &:active {
+        background-color: darken($toggle-colour--checked, f.$color-active-01);
+    }
+
+    .c-toggle-control {
+        @include toggle-transition(transform);
+
+        transform: $toggle-translation;
+    }
+}
+
+.c-toggle--modeRTL {
+    .c-toggle-control {
+        transform: $toggle-translation;
+    }
+}
+
+.c-toggle--modeRTL.c-toggle--checked {
+    .c-toggle-control {
+        @include toggle-transition(transform);
+
+        transform: translateX(0);
+    }
+}
```

Comment:
this is what I'm hoping we can avoid if we don't use a boolean for RTL

Reply:
Agree with this – would we be able to leverage the writing mode and use logical properties for this over a classname/prop?

(not sure how best to test this in storybook though – @jamieomaguire any ideas)

Reply:
you could quite trivially wrap the storybook template in a div with `dir="rtl` - I'm doing this on my PR soon :)